packages/editor-ui/src/views/NodeView.vue

```diff
@@ -2635,6 +2627,15 @@ export default defineComponent({
 						this.titleSet(workflow.name, 'IDLE');
 						await this.openWorkflow(workflow);
 						await this.checkAndInitDebugMode();
+
+						if (workflow.meta?.onboardingId) {
+							this.$telemetry.track(
+								`User opened workflow from ${workflow.meta.onboardingId} onboarding template`,
```

Comment:
Appcues will only show the modal once right? No matter if we triger the event again? I guess even if not it does not matter as we will redirect the user to this URL once, after the instance is provisioned.

Reply:
ya we will use the query param url which shows up only once